apps/judicial-system/web/src/routes/CourtOfAppeal/Overview/Overview.tsx

```diff
@@ -0,0 +1,277 @@
+import React, { useContext } from 'react'
+import { useIntl } from 'react-intl'
+import { useRouter } from 'next/router'
+import isValid from 'date-fns/isValid'
+import addDays from 'date-fns/addDays'
+
+import {
+  CaseDates,
+  CaseFilesAccordionItem,
+  FormContentContainer,
+  FormContext,
+  FormFooter,
+  InfoCard,
+  PageHeader,
+  PageLayout,
+  PdfButton,
+  RestrictionTags,
+  SignedDocument,
+  UserContext,
+} from '@island.is/judicial-system-web/src/components'
+import { AlertBanner, Box, Button, Text } from '@island.is/island-ui/core'
+import { core } from '@island.is/judicial-system-web/messages'
+import RulingDateLabel from '@island.is/judicial-system-web/src/components/RulingDateLabel/RulingDateLabel'
+import { capitalize, formatDate } from '@island.is/judicial-system/formatters'
+import Conclusion from '@island.is/judicial-system-web/src/components/Conclusion/Conclusion'
+import { CaseFileCategory } from '@island.is/judicial-system/types'
+import { useFileList } from '@island.is/judicial-system-web/src/utils/hooks'
+import * as constants from '@island.is/judicial-system/consts'
+
+import { courtOfAppealOverview as strings } from './Overview.strings'
+
+export const getStatementDeadline = (appealDate?: string) => {
```

Comment:
I wonder if we could move this logic to the transformer as well? Since we're already checking there whether the deadline has expired, to return the actual deadline as well since we also need to use it in other places

Reply:
Yes, I agree with you. I also think that appealDeadline should be there as well. I added a task to the inbox to fix this.

Reply:
Agreed, I also want to put a few more things there like appealedDate, I'll add to the asana task